api/src/service/domain/workflow/workflowitem_create.spec.ts

```diff
@@ -41,6 +41,7 @@ describe("Create workflowitem", () => {
 
     const result = await WorkflowitemCreate.createWorkflowitem(ctx, root, data, {
       workflowitemExists: async (_projectId, _subprojectId, _workflowitemId) => false,
+      userExists: async (_userId) => true,
```

Comment:
Please also add a new testcase that tests what you added, so if the user does not exist.

Also: what happens if you try this with a user that exists but is disabled? does this (and should this) still work?